src/workerd/api/tests/js-rpc-test.js

```diff
@@ -455,6 +456,23 @@ export class MyService extends WorkerEntrypoint {
       })()
     );
   }
+
+  testImportedWaitUntil() {
+    // Initialize globalWaitUntilPromise to a promise that will be resolved in a waitUntil task
+    // later on. We'll perform a cross-context wait to verify that the waitUntil task actually
+    // completes and resolves the promise.
+    let resolve;
+    globalWaitUntilPromise = new Promise((r) => {
+      resolve = r;
+    });
+
+    waitUntil(
```

Comment:
This isn't really testing anything: If you deleted the `waitUntil()` call (but kept the parameter computation) the test would still pass, because Durable Objects keep running for ~1 minute after the last request.

The only thing that seems worth testing here is that `waitUntil()` doesn't, like, throw an exception.

Reply:
My comment was wrong, which led to this confusion. I've updated the test. This was actually testing WorkerEntrypoint.